week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,7 +7,49 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
+1. select rate from rooms where rate > 100;
+   select *  from rooms where rate > 100;
 
+2.  select *, checkout_date - checkin_date as night  from reservations where checkin_date between '2020-09-01'and '2020-09-30' and checkout_date - checkin_date > 3;
+select *, checkout_date - checkin_date as night  from reservations where (extract(month from checkin_date)) = (extract(month from current_date)) and  checkout_date - checkin_date > 3;
```

Comment:
Good solution.  The first works fine but only now, the second works anytime but only as long as there are less than a year's worth of reservations.  Note that `extract(month from current_date)` gives the value 9 (no year component).  To be completely generic you'd need to use a more complex test.